sos/plugins/frr.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2007 Ranjith Rajaram <rrajaram@redhat.com>
```

Comment:
Minor nit, copyright line needs to be updated/corrected.

I guess this is a c'n'p error?

Reply:
@TurboTurtle thanks for heads-up. I've skipped the personal (c) lines now (e.g. like it is in acpid.py). I'd rather not claim copy monopoly on s/quagga/frr/g job ;). And hopefully it's better for eventual future c'n'p jobs ;).